saml/Dockerfile

```diff
@@ -5,5 +5,9 @@ RUN mvn clean install -DskipTests
 
 FROM eclipse-temurin:21-jre-alpine
 WORKDIR /app
+RUN mkdir -p /app/config
+RUN mkdir -p /app/saml_cert
 COPY --from=builder /app/target/*.jar app.jar
+COPY ./application.yml /app/config/application.yml
+COPY ./saml_cert/* /app/saml_cert/
```

Comment:
Docker build failed here because there isn't a folder called `saml_cert` in this repo.